### ISSUE TITLE
Virus??

### ISSUE THREAD

**Issue by Thamer44** [builtbybel/bloatbox#35]
what is this ????
![Web_Photo_Editor](https://user-images.githubusercontent.com/94402173/216448389-67dcf763-c353-4356-a372-c07983053225.jpg)
[bloatbox.zip](https://github.com/builtbybel/bloatbox/files/10573398/bloatbox.zip)

**Comment by IlyMooon** [comment]
It's totally fine, it's just an app that can delete windows apps

**Comment by marvellz** [comment]
@Thamer44 , No. It is worth considering if in the scan results you see a ratio of 25/60, but at the same time, if there are no leading vendors (Bitdefender, ESET, KAV, Dr.Web, Malwarebytes, AVG, Avast) in 25 detections, then there is no need to worry.
